### DISCUSSION TITLE
raceResult

### DISCUSSION THREAD

**Discussion by jpsaderi** [br-g/openf1#97] [Q&A] [answered]
Hi everyone,

I am trying to work on a project using your API, and my goal is to obtain the drivers and their respective positions at the end of the race. I tried using the 'positions' method and checking the last known position of a driver to determine their finishing position. After several attempts, I arrived at a result that could be correct for most races, but it doesn't work for all. For example, Spa-Francorchamps in 2024, where Russell was disqualified.

Is there any method where I can get "driver_number" and "position" at the end of the race?

Additionally, is there a way to determine the points earned per race?

**Comment by adityakotha03** [comment, marked as answer]
Unfortunately, currently, there isn't any way to fetch points earned per race from OpenF1. 

Use Ergast API! They have all this, we get the same from there [F1nsight](https://www.f1nsight.com/#/driver-comparison/max_verstappen/hamilton)

**Reply by NimmerrichterM** [reply]
Ergast is closing after this season.

**Reply by adityakotha03** [reply]
Other people will maintain their [version ](https://github.com/jolpica/jolpica-f1) of it, and there is also FastF1 if you want to use it.
